lib/manageiq/api/client/version.rb

```diff
@@ -1,7 +1,7 @@
 module ManageIQ
   module API
     class Client
-      VERSION = "0.3.5".freeze
+      VERSION = "0.3.6".freeze
```

Comment:
If you're going to change the version, can you also put that in the commit message as well?  I typical use `Release v0.3.6` in the commit message, and then in the body of the commit message I also put the but of the changelog as well, so people looking at the git history can see it front and center.